mocks/mocks.py

```diff
@@ -913,6 +913,31 @@ def create_cosmology(custom_cosmology_parameters,
     return cosmology
 
 
+def convert_mass_to_light(config,
+                          mass,
+                          band='g'):
+    """
+    Converts the mass of the galaxy to light.
+
+    Parameters
+    ----------
+    config : dic
+        A dictionary of configuration items.
+    mass : pynbody.array.SimArray
+        The mass of the simulation particles.
+    band : str, optional
+        The filter of the observation.
+
+    Returns
+    -------
+    astropy.units.Quantity
+        The light value of the simulation particles.
+    """
+    mass = ensure_unit(mass, u.M_sun)
```

Comment:
I am not sure about `ensure_unit` here.

Reply:
I think this is ok, `ensure_unit` is just doing `arg = arg.to(unit)` so making sure all mass is in units of M_sun and will convert it to that if not. I'm assuming `pynbody.array.SimArray` (never seen this before) is all in astropy mass units of some kind?

Reply:
`pynbody.array.SimArray` is the output of `Pynbody` which I use to read the data and the answer to your question is yes.

By not being sure about `ensure_unit`, I was trying to say that I am not sure if it is a good idea to use it here. Because if you check the function `read_gadget`, which reads the data and is happening before the current step, the units of mass is converted to `Msol`, you can find it in line 168.